src/helper/eye-dropper.js

```diff
@@ -0,0 +1,71 @@
+import paper from '@scratch/paper';
+
+const PAPER_WIDTH = 864;
+const PAPER_HEIGHT = 648;
```

Comment:
Where do these numbers come from?

Reply:
I had to look at the rendered canvas to see them – I can't find a reference anywhere in our code, but this appears to be something that paper is doing, and this is necessary to scale the loupe to make sure it's looking at the right part of the canvas when extracting a color.